scss/_custom-forms.scss

```diff
@@ -144,6 +144,7 @@
 
 .custom-radio {
   .custom-control-label::before {
+    // stylelint-disable-next-line property-blacklist
```

Comment:
I guess the custom radios should always be round, independent of `$enable-rounded`? That's why I didn't use the mixin here.